takt.py

```diff
@@ -95,15 +95,25 @@ def __init__(self, filename):
     def read(self, nrows=None):
         if not self.exists():
             raise ValueError(f"File {self.filename} does not exist.")
-        data = pd.read_csv(self.filename, nrows=nrows)
+        data = pd.read_csv(
+            self.filename,
+            nrows=nrows,
+            dtype=str,
```

Comment:
This was my first iteration, but then the file where the app logs the information has all the non-empty rows in the `notes` column filled with `nan` values; which is quite annoying. 

Tinkering with it, I ended up in the code you see in this PR.

This applies to your other comments.

Reply:
Mmm, thank you. However, it would be better if you could provide an example and justify the hacks with notes. It's important to understand why they are needed and if we find a better alternative, those lines could be removed as well.